File: LinqWhereNotInTest/LINQWhereNotInTestVB/Booking.vb
Imports System.Data.Entity

Public Class Booking
    Public Property ID As Integer
    Public Property MiscFieldForSample As String
End Class

Public Class ExclusionList
    Public Property ID As Integer
    Public Property PersonID As Integer
End Class

Public Class Ctx
    Inherits DbContext

    Public Sub New()
        Database.SetInitializer(Of Ctx)(New Init())
    End Sub
    Public Property Bookings As DbSet(Of Booking)
    Public Property Exclusions As DbSet(Of ExclusionList)
End Class

File: LinqWhereNotInTest/LINQWhereNotInTest/Booking.cs
using System;
using System.Collections.Generic;
using System.Data.Entity;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApplication14
{
    public class Booking
    {
        public int ID { get; set; }
        public string MiscFieldForSample { get; set; }

    }
    public class ExclusionList
    {
        public int ID { get; set; }
        public int PersonID { get; set; }
    }

    public class Ctx : DbContext
    {
        public Ctx()
        {
            Database.SetInitializer<Ctx>(new Init());
        }
        public DbSet<Booking> Bookings { get; set; }
        public DbSet<ExclusionList> Exclusions { get; set; }
    }
}
